contrib/cachetool.php

```diff
@@ -64,36 +66,51 @@
 set('cachetool_options', function () {
     $options = get('cachetool');
     $fullOptions = get('cachetool_args');
+    $return = [];
 
     if (strlen($fullOptions) > 0) {
-        $options = "{$fullOptions}";
+        $return = ["{$fullOptions}"];
     } elseif (strlen($options) > 0) {
-        $options = "--fcgi={$options}";
+        if (!is_array($options)) {
```

Comment:
You can shorten that by just casting it to array - if it is a string it becomes ['string'], if it is array it stays as it is :-)